Frogies-Project/Assets/Scripts/Fighting/BasicAttacker.cs

```diff
@@ -45,7 +46,8 @@ public bool CanPerformAttack(int attackIndex)
         {
             return attackIndex != -1
                    && _attackRechargeTimer <= 0
-                   && _enduranceSystem.CheckEnduranceAbility(_attacksData.Attacks[attackIndex].enduranceCost);
+                   && _enduranceSystem.CheckEnduranceAbility(_attacksData.Attacks[attackIndex].enduranceCost)
+                   && !GlobalSceneManager.Instance.PlayerData.DirectionalMover.IsDashing;
```

Comment:
Enemies also use BasicAttacker. In this case enemies can't attack while player is dashing